shadows/framework/src/main/java/org/robolectric/shadows/ShadowScrollView.java

```diff
@@ -16,4 +16,9 @@ protected void smoothScrollTo(int x, int y) {
   protected void smoothScrollBy(int x, int y) {
     scrollBy(x, y);
   }
+
+  @Implementation
+  protected void scrollTo(int x, int y) {
```

Comment:
Hi @brettchabot , thanks your review and replay. In [AOSP `ScrollView`'s implementation of `smoothTo`](https://android.googlesource.com/platform/frameworks/base/+/master/core/java/android/widget/ScrollView.java#1782), it will check the child count, and only call the super `scrollTo`, `View.scrollTo`, when the child count is larger than 0, and new scroll location is not equals to the location calculated by `clamp` method based on the first child of `ScrollView`. If we want to test logic with `ScrollView.scrollTo`, we should add child for `ScrollView`, and ensure the new location can pass the checking of `clamp`. It's more complicated than test logic with `ScrollView.smoothScrollTo`, that uses `ShadowView.scrollTo` to set the new scroll location to `View` directly. So this pr adds `ShadowScrollView.scrollTo` use `ShadowView.scrollTo` directly too to make test with `ScrooView.scrollTo` more simply.